### DISCUSSION TITLE
Can someone give a guide on how I can BuildWSA myself?

### DISCUSSION THREAD

**Discussion by AbuuUbaydah** [LSPosed/MagiskOnWSALocal#336] [Q&A]
Can someone give a guide on how I can BuildWSA myself?

It is no longer available under the actions tab, and the instructions attached to the repo are not clear enough for me.

**Comment by shrublet** [comment]
I'm gonna explain the simplest way to do this on Windows. Install WSL (Windows Subsystem for Linux) and Ubuntu from the Windows Store. Open the Ubuntu app which should open in a Terminal window. Clone the repository in Ubuntu (this is what it means by cloning locally, not forking the repo) by running `git clone https://github.com/LSPosed/MagiskOnWSALocal.git` in whatever directory you want it in. The rest *should* be self-explanatory enough. Just read the guide step-by-step while you're doing it.
